spec/models/event_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Event, type: :model do
```

Comment:
@sharadalt If you'd like the practice writing tests, consider writing specs for the Event model (testing its association and validations) and write specs for the future functionality you add to blocmetrics from now on.